src/main/java/com/avaya/letscode/Cart.java

```diff
@@ -7,11 +7,11 @@
 
 public class Cart {
     //List of items in the cart with respective unit quantity
-    Map<Product, CartItem> items;       
+    private Map<Product, CartItem> items;       
```

Comment:
This can be marked final as we don't expect it to get assigned a new Map once created in the constructor.
//
 private final Map<Product, CartItem> items;